app/controllers/api/v1/articles_controller.rb

```diff
@@ -20,11 +20,13 @@ def create
       end
 
       def destroy
+        authorize(article, policy_class: ArticlePolicy)
```

Comment:
Also, I think we should put this authorization inside the service otherwise we are doing Article.find(id) twice and I am not  fan of that

Reply:
this ```authorize article``` and ```Also, I think we should put this authorization inside the service otherwise we are doing Article.find(id) twice and I am not fan of that``` didn't work so i did this way